components/hab/mac/mac-build.sh

```diff
@@ -30,33 +30,34 @@ install_if_missing() {
     local formula="$pkg"
   fi
 
-  echo $(sudo -u ${SUDO_USER} brew list --versions ${pkg})
+  sudo -u "${SUDO_USER}" brew list --versions "${pkg}"
 
-  if [[ $(sudo -u ${SUDO_USER} brew list --versions $pkg | wc -l) -eq 0 ]]; then
+  if [[ $(sudo -u "${SUDO_USER}" brew list --versions "$pkg" | wc -l) -eq 0 ]]; then
     info "Installing missing Homebrew package $formula"
-    sudo -u $SUDO_USER brew install $formula
+    sudo -u "$SUDO_USER" brew install "$formula"
```

Comment:
Why is `SUDO_USER` on line 35 inside both double quotes and braces, but here on line 37, just quotes?

Reply:
The braces don't disable globbing or word-splitting. They can be nice visually, but our codebase is wildly inconsistent about whether or not they're used, so I left the code either with or without braces as I found it and just added the double quotes where necessary to satisfy `shellcheck`

Reply:
Fair enough.  The OCD part of me wants consistency, but I can appreciate your desire to change as little as possible to get things in a passable state.